kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/ClientConfigurator.java

```diff
@@ -0,0 +1,36 @@
+package org.arquillian.cube.kubernetes;
+
+import io.fabric8.kubernetes.api.builder.v3_1.TypedVisitor;
+import io.fabric8.kubernetes.clnt.v3_1.ConfigBuilder;
+import org.arquillian.cube.impl.util.Strings;
+import org.arquillian.cube.kubernetes.api.Configuration;
+
+public class ClientConfigurator {
```

Comment:
I don't see any Client Configuration, where as I can see here `ClientConfigBuilder`.
This name is not relevant to what is it doing in the class. Also How about making this class to use something like
```java
new ClientConfigBuilder()
  .configuration(config)
  .build()
```